src/Class7/DevidedNumbers.java

```diff
@@ -0,0 +1,26 @@
+package Class7;
+
+import java.sql.Array;
+//1. Напишіть метод, який ділить кожен елемент масива на задане число і виводить на
+//екран новий масив елементів із поділеними елементами попереднього. Додайте
+//опрацювання виняткових ситуацій
+
+public class DevidedNumbers {
+    public static double[] devideNumbers(double[] array, double number) {
+        double[] newArray = new double[array.length];
+        for (int i = 0; i < array.length; i++) {
+            newArray[i] = array[i] / number;
```

Comment:
it's needed to handle exception in case /0

Reply:
public static void divideArrayElements(int[] array, int divisor) {
        int[] dividedArray = new int[array.length];

        try {
            for (int i = 0; i < array.length; i++) {
                dividedArray[i] = array[i] / divisor;
            }
            System.out.println("Divided array elements:");
            for (int num : dividedArray) {
                System.out.println(num);
            }
        } catch (ArithmeticException e) {
            System.out.println("Error: Division by zero is not allowed.");
        } finally {
            System.out.println("Finally block executed.");
        }
    }